skbio/sequence/distance.py

```diff
@@ -113,3 +104,54 @@ def hamming(seq1, seq2):
         distance = scipy.spatial.distance.hamming(seq1.values, seq2.values)
 
     return float(distance)
+
+
+@experimental(as_of='0.4.2')
+def kmer_distance(seq1, seq2, k=3, overlap=True):
+    """Compute the kmer distance between a pair of sequences
+    Parameters
+    ----------
+    seq1 : skbio.Sequence
+    seq2 : skbio.Sequence
+    k : int, optional
+        The word length.
+    overlap : bool, optional
+        Defines whether the k-words should be overlapping or not
+        overlapping.
+    Returns
+    -------
+    float
+        Fraction of the set of k-mers from both seq1 and
+        seq2 that are unique to either seq1 or
+        seq2.
+    Raises
+    ------
+    ValueError
+        If k < 1.
+    Notes
+    -----
+    k-mer counts are not incorporated in this distance metric.
+    """
+    _check_seqs(seq1, seq2)
+    seq1_kmers = set(map(str, seq1.iter_kmers(k, overlap)))
```

Comment:
Can you make both of the `iter_kmers` calls pass `overlap=overlap`? That'll future-proof the code in case `iter_kmers` changes its parameter order.